templates/master/00-master/_base/files/etc-kubernetes-manifests-etcd-member.yaml

```diff
@@ -72,7 +72,7 @@ contents:
           mountPath: /etc/kubernetes/kubeconfig
       containers:
       - name: etcd-member
-        image: "quay.io/coreos/etcd:v3.3.10" ## FIXME(abhinav): these images should be replacable by release image.
+        image: "{{.Images.setupEtcdEnv}}"
```

Comment:
Should this be `.Images.etcd` ?

Reply:
fixed with https://github.com/openshift/machine-config-operator/pull/343/commits/e73171b74d6d6296d10d6015ac58330e615b9679